Added links of webdriverio files and renames class names. (#11)
oppia/oppia-web-developer-docs: WebdriverIO.md

Then you can select Element B with this selector chain:

```js
var elemB = $('.e2e-test-elem-a').$('#elem-b');
```

Notice that the top of the chain, where we select Element A, uses method 1.

Sometimes you need to distinguish between several different elements which all look the same to element selectors. You can iterate over all the elements to find the right one. For example, suppose we want to click on the button to open a topic, where the button text is the topic name. We could find the right button like this:

```js
var buttonsSelector = function() {
  return $$('.e2e-test-button');

> }

Reviewer: Wouldn't it be good if you explain this too a bit in terms of flakiness like why we need a function for all selector when it's declared at the top?

Author: Done.

}
```

> As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as it will try to find the element as soon as the test will run, and this will cause the test to fail.

Reviewer: Can you please also add the consequences of this sentence? From this line, my understanding is that it's good only right if the all selector element is found first when the test will run. But my question is what's the difference between declaring it at the top using await and then accessing it using the defined variable in comparision to declaring a function at the top and then calling the function in the tests in order to access the element.

`it will try to find the element as soon as the test will run`

Author: Updated the comment PTAL.

> As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as `await` can only be used inside `async` function and if we will convert the function to `async` we cannot use it as a constructor for defining page objects.

Reviewer: ```suggestion
As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as `await` can only be used inside `async` function and if we convert the function to `async` we cannot use it because of [this](https://stackoverflow.com/questions/51694221/not-a-constructor-with-async-function#:~:text=When%20you%20use%20the%20async,can't%20be%20a%20constructor) error.
```

Author: Done.

> 

Reviewer: >and we can use `await` only inside async function

I am not able to understand this. I think we use await prefix inside async function only, right? For eg see openTopic is an async function only. Hence we are able to use await.

Author: Yes, but we need to place the selector at top, and there we cannot use await.

Reviewer: My comment was why we can't use `await $$selector` at the top instead of having a function for it and then calling it because theoritically both would mean the same. This is in reference to what you said above i.e we need an await for $$ selector.

Author: Is it fine now?

Reviewer: PTAL at https://github.com/oppia/oppia-web-developer-docs/pull/11#discussion_r922803180

Except for cases where an element selector is crafted dynamically, all element selectors should be at the top of a utility file. There should be no element selectors in suite files.

> Note: We need to use `await` prefix while using `$$` selector in order to avoid flakiness of tests.

Reviewer: Also, mention the case where we don't need it.

Author: Done.
